index.html

```diff
@@ -0,0 +1,200 @@
+          </h3>
+        </div>
+      </div>
+    </section>
+
+    <section class="grid bg-pattern-1 bg-dark white" id="dates">
+      <div class="container f-col ai-center m-section">
+        <h2 class="white fs-30">
+          Global Schedule<br />
+          <hr class="line-orange" />
+        </h2>
+
+        <div class="f-row w-100 jc-between g-3 m-40" id="dates__container">
+          <div class="f-row g-3 w-100 dates__paired">
+            <a class="w-100" href="#">
+              <div class="box transparent h-100 f-col ai-center border-white-hover g-17">
+                <i class="bi bi-globe-americas fs-40"></i>
+                <h3 class="orange fs-25 ta-center">Americas</h3>
+                <p class="ta-center fs-18">
+                  Featuring captivating performances in the United States,
+                  Canada, Brazil, Argentina, and much more.
+                </p>
+              </div>
+            </a>
+
+            <a class="w-100" href="#">
+              <div class="box transparent h-100 f-col ai-center border-white-hover g-17">
+                <i class="bi bi-globe-europe-africa fs-40"></i>
+                <h3 class="orange fs-25 ta-center">Africa & Europe</h3>
+                <p class="ta-center fs-18">
+                  Showcasing unforgettable concerts in France, South Africa,
+                  Germany, Morocco, and much more.
+                </p>
+              </div>
+            </a>
+          </div>
+          <div class="f-row g-3 w-100 dates__paired">
+            <a class="w-100" href="#">
+              <div class="box transparent h-100 f-col ai-center border-white-hover g-17">
+                <i class="bi bi-globe-asia-australia fs-40"></i>
+                <h3 class="orange fs-25 ta-center">Asia & Australia</h3>
+                <p class="ta-center fs-18">
+                  Thrilling performances in Japan, Australia, China, India,
+                  and much more.
+                </p>
+              </div>
+            </a>
+
+            <a class="w-100" href="#">
+              <div class="box transparent h-100 f-col ai-center border-white-hover g-17">
+                <i class="bi bi-globe fs-40"></i>
+                <h3 class="orange fs-25 ta-center">Globe</h3>
+                <p class="ta-center fs-18">
+                  Check the complete list of global concerts here!
+                </p>
+              </div>
+            </a>
+          </div>
+        </div>
+        <a class="ta-center underline-hover fs-18" href="#">SEE THE WHOLE PROGRAM</a>
+      </div>
+    </section>
+    <section class="grid" id="band">
+      <div class="container dark f-col ai-center m-section g-50">
+        <h2 class="fs-30 ta-center">
+          Band Members<br />
+          <hr class="line-orange" />
+        </h2>
+        <div class="container f-col g-70 f-wrap" id="band__band-members-cards">
+
+        </div>
+      </div>
+    </section>
+    <section class="grid bg-grey" id="support">
```

Comment:
Great design @grauJavier 🤝

- [ ] As stated in the project requirements, kindly ensure your design conforms to the design provided.
- Your design
![Your footer](https://github.com/grauJavier/capstone-project/assets/63917725/e529bac4-bb83-47f0-ae87-0f1c3c5a3b61)
- Given design
![Given footer](https://github.com/grauJavier/capstone-project/assets/63917725/4e59725e-dc09-42b6-b5ff-315e66d83ef0)